### PR TITLE
[KFTO-SDK] Add doc on customising base images for Train API

### PR DESCRIPTION
**Pull request by varshaprasad96**
Follow up from https://github.com/kubeflow/training-operator/pull/2261 as this is a user facing change.

### CODE REVIEW

File: content/en/docs/components/training/user-guides/fine-tuning.md [hunk 1]
```diff
@@ -87,6 +87,9 @@ TrainingClient().train(
 After you execute `train`, the Training Operator will orchestrate the appropriate PyTorchJob resources
 to fine-tune the LLM.
 
+**NOTE**:
+Platform engineers can customize the storage initializer and trainer images by setting the `STORAGE_INITIALIZER_IMAGE` and `TRAINER_TRANSFORMER_IMAGE` environment variables before executing the `train` command.
```
Comment (andreyvelich) [inline content/en/docs/components/training/user-guides/fine-tuning.md:91]: Thank you for adding this!
Please can you add a new section in this doc, so we can reference users to that link

## Use custom images for the Fine-Tuning API

Comment (andreyvelich) [inline content/en/docs/components/training/user-guides/fine-tuning.md:91]: Also, please add a simple example to explain how to modify it. For example:
```python
os.environ["STORAGE_INITIALIZER_IMAGE"] = "docker.io/custom-storage-initiailizer`
os.environ["TRAINER_TRANSFORMER_IMAGE"] = "docker.io/custom-trainer-transformer`

TrainingClient().train(...)

```

**Review by andreyvelich — COMMENTED** [review]
Thanks for adding this @varshaprasad96!

**Review by Arhell — COMMENTED** [review]
/ok-to-test

**Comment by varodrig** [comment]
> Thanks for adding this @varshaprasad96!

@andreyvelich  is this PR ready to be approved?

**Review by andreyvelich — COMMENTED** [review]
Yes, we can merge it.
Thanks for the contribution @varshaprasad96!
/lgtm
/approve

**Pull request merged**